src/lib/dateTime.ts

```diff
@@ -0,0 +1,56 @@
+import moment = require('moment')
+
+export class DateTime {
+	private static readonly numberToMonthName = require('number-to-date-month-name')
+	private static readonly convert = require('convert-seconds')
+	private static readonly isoRegex = new RegExp(
+		'^(-)?P(?:(\\d+)Y)?(?:(\\d+)M)?(?:(\\d+)D)?' + '(T(?:(\\d+)H)?(?:(\\d+)M)?(?:(\\d+(?:\\.\\d+)?)S)?)?$'
```

Comment:
Moment already has iso range for validation and formatting. Why are we redefining this?

Reply:
please see comment below - created a task to revisit that part